examples/iot-nirvana/README.md

```diff
@@ -41,23 +41,39 @@ In order to run the simulation in ideal conditions, with 10 virtual machines,
 please request an increase of your CPU quota to 80 vCPU. This is however
 optional.
 
+Set environment variables used in setting up this tutorial edit the defaults if you wish (note not all products may be in all regions) :
```

Comment:
I would suggest something along the lines of "Create the environment variables that will be used throughout this tutorial. You can edit the default values provided below, however please note that not all products may be available in all regions:"